soil-gas/README.md

```diff
@@ -0,0 +1,44 @@
+
+Soil gas flux measurements were made regularly for monitoring changes at Whakaari White Island until the 2019 eruption. It that case, pegs were used to mark measurement points, so measurements could be made at exactly the same positions each time. Those data constitute a time-series, and are therefore available through [Tilde](https://tilde.geonet.org.nz/).
+
+Soil gas flux measurements have also been made at Raoul Island, Ngauruhoe, Tongariro, and Moutohorā Whale Island volcanoes. In these cases, pegs were not used to mark measurement points, so measurements could not be made at exactly the same positions each time. Instead, measurements are made in broadly the same areas, but subject to constraints such as access and time availability, which will vary from survey to survey. For this reason, making the data available through Tilde is not an option. Those data are available through this repository.
+
+## Data
+The location of each gas flux measurement is measured by a phone with a built-in GPS and recorded alongside the flux measured. The ground temperature at a depth of 10 cm is measured immediately after the soil gas measurement by inserting a thermocouple in the ground at the middle where the chamber was located.
+
+Measurement locations are provided in the [New Zealand Transverse Mercator 2000 (NZTM2000) grid](https://www.linz.govt.nz/guidance/geodetic-system/coordinate-systems-used-new-zealand/projections/new-zealand-transverse-mercator-2000-nztm2000), rather than latitude and longitude in the [World Geodetic System 1984 (WGS84)](https://www.linz.govt.nz/guidance/geodetic-system/coordinate-systems-used-new-zealand/geodetic-datums/world-geodetic-system-1984-wgs84) coordinate system that is commonly used for GeoNet spatial data.
+
+## Description of Files and Data Fields
+Each file in this repository contains the data collected from one survey of one volcanic area. Files are in CSV (comma-separated values) format.
+
+### Filenames
+A filename consists of three parts:
+- The volcano name, e.g "MoutohoraWhaleIsland". If both Te Reo and English names are commonly used for the volcano, the Te Reo name is followed by the English name. There are no spaces in the volcano name, and no macrons are used for Te Reo names.
+- The date the data were collected, e.g. "2025-01-29". This uses local time (NZST or NZDT). The date is separated from the volcano name by an underscore “\_”.
+- A “csv” suffix. This indicates a CSV file format. The suffix is separated from the date by a dot “.”.
+If measurements were carried out over more than one day, but are considered by the VMG  to represent a single dataset, they are provided as a single file without highlighting which observations were made on which day. However, the date part of the filename shows both measurement dates.
```

Comment:
Initially, this confused me. Seemed like the convention was for files to reflect local time date samples but here it says multiple days could be in one file. How the date part of the filename can show both measurement dates? I will need to see an example.

Reply:
I have seen the example. I think the convention works but needs to be spelled out in the readme. Probably just on row 21 after ` The date is separated from the volcano name by an underscore “\_”.` with something like "If the data are sampled over multiple local time dates, the convention `start-date_end-date` is used instead of `sample-date`."